xyo-android-sample/src/main/res/values/strings.xml

```diff
@@ -12,5 +12,5 @@
     <string name="public_key">Public Key: </string>
     <string name="detected_devices">Detected Devices:</string>
     <string name="detected_xyo_devices">Detected XYO Devices:</string>
-    <string name="nearby_xyo_devices">Nearby XYO Devices:</string>
+    <string name="nearby_xyo_devices">Interactions:</string>
```

Comment:
Can you explain this change? Thanks

Reply:
This value is increasing over time with more bound witness interactions, so, i've concluded that this should be the number of interactions during the app running (background or first plan).

Reply:
Ah, I'll have to take a closer look at this. Thanks for all of the responses!

Reply:
Ok! I've just forgot... The test environment is my own android phone (Moto G8 Plus android Pie).

Reply:
I'm just thinking: it is possible to put an status of Battery Level in the interface? Battery level for SentinelX devices... I've seen that it has an battery service enabled, but is not acquiring the data to the interface.